enhancements/olm/olm-managed-operator-metrics.md

```diff
@@ -110,26 +106,58 @@ A `ServiceMonitor` object must be created in the namespace mentioned above and p
 
 ##### RBAC Requirements
 
-A role/rolebinding must be created that provides the Prometheus Operator ServiceAccount with the appropriate RBAC privileges to discover the `ServiceMonitor` and `PrometheusRule` resources in the newly created namespace.
+A role and rolebinding must be created that provides the OpenShift Monitoring `prometheus-k8s` ServiceAccount with the appropriate RBAC privileges to discover the `Services` in your operator's namespace. An example of the required RBAC can be seen below:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: placeholder
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - endpoints
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: prometheus-k8s
+  namespace: placeholder
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: prometheus-k8s
+subjects:
+- kind: ServiceAccount
+  name: 'prometheus-k8s',
+  namespace: 'openshift-monitoring',
+```
 
 #### Fulfilling Namespace and RBAC Requirements
 
-OLM itself will not be responsible for fulfilling the Namespace and RBAC requirements. Instead, OLM will define how operator authors can provide OpenShift Console with the information required to fulfill these requirements.
+The OpenShift Monitoring Requirements mentioned above can be met in one of two ways:
 
-OLM [building-your-csv](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md) documentation will be updated to describe two new supported annotations:
+- By using annotations that instructs OpenShift Console to generate the required Namespace and RBAC Requirements
+- By packaging the RBAC [mentioned above](#####rbac-requirements) within the Operator Bundle and having a cluster admin install the operator in a namespace that is both prefixed with `openshift-` and has the `openshift.io/cluster-monitoring=true` label.
```

Comment:
[food for thought](https://hackmd.io/mwY2A7V4RjOLbetL--Uk-w).

Reply:
> in a namespace that is both prefixed with `openshift-`
I guess this is too restrictive if mandatory.